pkg/telemetry/tracerexporters/gcp.go

```
@@ -0,0 +1,31 @@
+package tracerexporters
+
+import (
+	"os"
+
+	texporter "github.com/GoogleCloudPlatform/opentelemetry-operations-go/exporter/trace"
+	sdktrace "go.opentelemetry.io/otel/sdk/trace"
+)
+
+// NewGCP creates a new Google Cloud tracer with optional headers for credentials and project ID.
+func NewGCP(url string, insecure bool, headers map[string]string) (sdktrace.SpanExporter, error) {
+
```

golangci-lint

File is not `gofumpt`-ed with `-extra`

(gofumpt)